packages/themes/blue-jeans/.stylelintignore

```diff
@@ -0,0 +1 @@
+src/fonts/family/base64-fonts.scss
```

Comment:
already excluded by the command, both should not be needed

Reply:
I changed the command as I thought this way is much cleaner : command should target folders, configuration should exclude rules and files. [https://stylelint.io/user-guide/ignore-code/#files-entirely](https://stylelint.io/user-guide/ignore-code/#files-entirely)

Reply:
Then you should update the command to remove the exclusion